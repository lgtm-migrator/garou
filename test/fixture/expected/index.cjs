// @ts-nocheck
/* This is a comment with no whitespace at the beginning */
'use strict';

import 'll';

import { foo, z } from 'bar';
import n, { kde } from 'nn';
import c from 'vvv';

import ddd from './ss';

const k = [];

class A {
  constructor() {}

  a() {}
}

function a() {}

function x() {
  if (xsd) {
  }
}

const y = [];

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"',
};

export { e } from '../..';

export { g } from '.';

export { jkjk, xxxx } from 'c';

export { h } from 'x';

if (ytt) {
} else {
}

export function aaaaa() {}

if (xsd) {
}
