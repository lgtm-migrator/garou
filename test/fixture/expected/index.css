/* stylelint-disable block-no-empty, unit-no-unknown */
body {
  width: 0;
  height: 2;
}

div::placeholder {
  display: block;
  appearance: flex;
  color: #ffffff;
  opacity: 0.5;
}

/* comment */
div::before {
  --foo: hsl(198deg 28% 50%);

  color: rgba(255, 255, 255, 0.5);

  @media (min-device-pixel-ratio > 1) {
  }
}

@keyframes {
  0% {
    transform: calc(1px + 2px);
  }
}
