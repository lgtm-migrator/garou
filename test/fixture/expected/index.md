# f

```js
/* This is a comment with no whitespace at the beginning */
'use strict';

import 'll';

import { foo, z } from 'bar';
import n, { kde } from 'nn';
import c from 'vvv';

import ddd from './ss';

const k = [];

class A {
  constructor() {}

  a() {}
}

function a() {}

function x() {
  if (xsd) {
  }
}

const y = [];

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"'
};

export { e } from '../..';

export { g } from '.';

export { jkjk, xxxx } from 'c';

export { h } from 'x';

if (ytt) {
} else {
}

export function aaaaa() {}

if (xsd) {
}
```

```cjs
/* This is a comment with no whitespace at the beginning */
'use strict';

import 'll';

import { foo, z } from 'bar';
import n, { kde } from 'nn';
import c from 'vvv';

import ddd from './ss';

const k = [];

class A {
  constructor() {}

  a() {}
}

function a() {}

function x() {
  if (xsd) {
  }
}

const y = [];

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"'
};

export { e } from '../..';

export { g } from '.';

export { jkjk, xxxx } from 'c';

export { h } from 'x';

if (ytt) {
} else {
}

export function aaaaa() {}

if (xsd) {
}
```

```mjs
/* This is a comment with no whitespace at the beginning */
'use strict';

import 'll';

import { foo, z } from 'bar';
import n, { kde } from 'nn';
import c from 'vvv';

import ddd from './ss';

const k = [];

class A {
  constructor() {}

  a() {}
}

function a() {}

function x() {
  if (xsd) {
  }
}

const y = [];

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"'
};

export { e } from '../..';

export { g } from '.';

export { jkjk, xxxx } from 'c';

export { h } from 'x';

if (ytt) {
} else {
}

export function aaaaa() {}

if (xsd) {
}
```

```jsx
/* This is a comment with no whitespace at the beginning */
'use strict';

import 'll';

import { foo, z } from 'bar';
import n, { kde } from 'nn';
import c from 'vvv';

import ddd from './ss';

const k = [];

class A {
  constructor() {}

  a() {}
}

function a() {}

function x() {
  if (xsd) {
  }
}

const y = [];

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"'
};

export { e } from '../..';

export { g } from '.';

export { jkjk, xxxx } from 'c';

export { h } from 'x';

if (ytt) {
} else {
}

export function aaaaa() {}

if (xsd) {
}

const d = (
  <R a={1} df={true} prop="Hello world">
    {'Hello world'}
  </R>
);
```

```ts
/* This is a comment with no whitespace at the beginning */
'use strict';

import 'll';

import { foo, z } from 'bar';
import n, { kde } from 'nn';
import c from 'vvv';

import ddd from './ss';

const k = [];

class A {
  constructor() {}

  a() {}
}

function a(t: string) {}

function x() {
  if (xsd) {
  }
}

const y = [];

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"'
};

export { e } from '../..';

export { g } from '.';

export { jkjk, xxxx } from 'c';

export { h } from 'x';

if (ytt) {
} else {
}

export function aaaaa() {}

if (xsd) {
}
```

```tsx
/* This is a comment with no whitespace at the beginning */
'use strict';

import 'll';

import { foo, z } from 'bar';
import n, { kde } from 'nn';
import c from 'vvv';

import ddd from './ss';

const k = [];

class A {
  constructor() {}

  a() {}
}

function a(t: string) {}

function x() {
  if (xsd) {
  }
}

const y = [];

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"'
};

export { e } from '../..';

export { g } from '.';

export { jkjk, xxxx } from 'c';

export { h } from 'x';

if (ytt) {
} else {
}

export function aaaaa() {}

if (xsd) {
}

const d = (
  <R a={1} df={true} prop="Hello world">
    {'Hello world'}
  </R>
);
```

```vue
<template>
  <!-- jj -->
  <!-- ✗ BAD -->
  <!--
    ✗ BAD
    d
  -->
  <img id="" class="" data-id="" />
</template>

<script>
/* This is a comment with no whitespace at the beginning */
'use strict';

import 'll';

import { foo, z } from 'bar';
import n, { kde } from 'nn';
import c from 'vvv';

import ddd from './ss';

const k = [];

class A {
  constructor() {}

  a() {}
}

function a() {}

function x() {
  if (xsd) {
  }
}

const y = [];

export default {
  name: 'd',
  data() {},
  a: `${5 ** 2}`,
  kk: '444"'
};

export { e } from '../..';

export { g } from '.';

export { jkjk, xxxx } from 'c';

export { h } from 'x';

if (ytt) {
} else {
}

export function aaaaa() {}

if (xsd) {
}
</script>

<style>
/* stylelint-disable block-no-empty, unit-no-unknown */
body {
  width: 0;
  height: 2;
}

div::placeholder {
  display: block;
  appearance: flex;
  color: #ffffff;
  opacity: 0.5;
}

/* comment */
div::before {
  --foo: hsl(198deg 28% 50%);

  color: rgba(255, 255, 255, 0.5);

  @media (min-device-pixel-ratio > 1) {
  }
}

@keyframes {
  0% {
    transform: calc(1px + 2px);
  }
}
</style>

<style lang="less">
body {
  width: 0;
  height: 2;

  div::placeholder {
    display: block;
    appearance: flex;
    color: #ffffff;
    opacity: 0.5;
  }
}

/* comment */
div::before {
  --foo: hsl(198deg 28% 50%);

  color: rgba(255, 255, 255, 0.5);

  @media (min-device-pixel-ratio > 1) {
  }
}

@keyframes {
  0% {
    transform: calc(1px + 2px);
  }
}
</style>

<style lang="scss">
body {
  width: 0;
  height: 2;

  div::placeholder {
    display: block;
    appearance: flex;
    color: #ffffff;
    opacity: 0.5;
  }
}

/* comment */
div::before {
  --foo: hsl(198deg 28% 50%);

  color: rgba(255, 255, 255, 0.5);

  @media (min-device-pixel-ratio > 1) {
  }
}

@keyframes {
  0% {
    transform: calc(1px + 2px);
  }
}

@if $r {
  $d: 3;

  // s
  @include main();
}
</style>
```

```html
<!DOCTYPE html>
<html>
  <head>
    <style>
      body {
        width: 0;
        height: 2;
      }

      div::placeholder {
        display: block;
        appearance: flex;
        color: #ffffff;
        opacity: 0.5;
      }

      /* comment */
      div::before {
        --foo: hsl(198deg 28% 50%);

        color: rgba(255, 255, 255, 0.5);

        @media (min-device-pixel-ratio > 1) {
        }
      }

      @keyframes {
        0% {
          transform: calc(1px + 2px);
        }
      }
    </style>
  </head>
  <body id="" class="">
    <script>
      /* This is a comment with no whitespace at the beginning */
      import { foo, z } from 'bar';

      class A {
        constructor() {}

        a() {}
      }

      function a() {}

      export default {
        data() {},
        a: `${5 ** 2}`,
        kk: '444"'
      };
    </script>
  </body>
</html>
```

```css
body {
  width: 0;
  height: 2;
}

div::placeholder {
  display: block;
  appearance: flex;
  color: #ffffff;
  opacity: 0.5;
}

/* comment */
div::before {
  --foo: hsl(198deg 28% 50%);

  color: rgba(255, 255, 255, 0.5);

  @media (min-device-pixel-ratio > 1) {
  }
}

@keyframes {
  0% {
    transform: calc(1px + 2px);
  }
}
```

```scss
body {
  width: 0;
  height: 2;

  div::placeholder {
    display: block;
    appearance: flex;
    color: #ffffff;
    opacity: 0.5;
  }
}

/* comment */
div::before {
  --foo: hsl(198deg 28% 50%);

  color: rgba(255, 255, 255, 0.5);

  @media (min-device-pixel-ratio > 1) {
  }
}

@keyframes {
  0% {
    transform: calc(1px + 2px);
  }
}

@if $r {
  $d: 3;

  // s
  @include main();
}
```

```less
body {
  width: 0;
  height: 2;

  div::placeholder {
    display: block;
    appearance: flex;
    color: #ffffff;
    opacity: 0.5;
  }
}

/* comment */
div::before {
  --foo: hsl(198deg 28% 50%);

  color: rgba(255, 255, 255, 0.5);

  @media (min-device-pixel-ratio > 1) {
  }
}

@keyframes {
  0% {
    transform: calc(1px + 2px);
  }
}
```
