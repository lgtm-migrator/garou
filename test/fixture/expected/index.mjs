// @ts-nocheck
/* This is a comment with no whitespace at the beginning */
import { foo, z } from 'bar';

class A {
  constructor() {}

  a() {}
}

function a() {}

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"',
};
