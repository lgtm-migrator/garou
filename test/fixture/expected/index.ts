// @ts-nocheck
/* This is a comment with no whitespace at the beginning */
import { foo, z } from 'bar';

class A {
  constructor() {}

  a() {}
}

function a(t: string) {}

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"',
};
