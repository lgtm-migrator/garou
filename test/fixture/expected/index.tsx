// @ts-nocheck
/* This is a comment with no whitespace at the beginning */
import { foo, z } from 'bar';

class A {
  constructor() {}

  a() {}
}

function a(o: string) {}

export default {
  data() {},
  a: `${5 ** 2}`,
  kk: '444"',
};

const d = (
  <R a={1} df={true} prop="Hello world">
    {'Hello world'}
  </R>
);
