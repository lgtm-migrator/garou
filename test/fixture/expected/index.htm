<!DOCTYPE html>
<html>
  <head>
    <style>
      /* stylelint-disable block-no-empty, unit-no-unknown */
      body {
        width: 0;
        height: 2;
      }

      div::-moz-placeholder {
        display: block;
        -moz-appearance: -webkit-flex;
        color: #ffffff;
        opacity: 0.5;
      }

      /* comment */
      div::before {
        --foo: hsl(198deg 28% 50%);

        color: rgba(255, 255, 255, 50%);

        @media (-webkit-min-device-pixel-ratio > 1) {
        }
      }

      @-webkit-keyframes {
        0% {
          -webkit-transform: calc(1px + 2px);
        }
      }
    </style>
  </head>
  <body class="" id="">
    <script>
      /* This is a comment with no whitespace at the beginning */
      'use strict';

      import 'll';

      import { foo, z } from 'bar';
      import n, { kde } from 'nn';
      import c from 'vvv';

      import ddd from './ss';

      const k = [];

      class A {
        constructor() {}

        a() {}
      }

      function a() {}

      function x() {
        if (xsd) {
        }
      }

      const y = [];

      export default {
        data() {},
        a: `${5 ** 2}`,
        kk: '444"',
      };

      export { e } from '../..';

      export { g } from '.';

      export { jkjk, xxxx } from 'c';

      export { h } from 'x';

      if (ytt) {
      } else {
      }

      export function aaaaa() {}

      if (xsd) {
      }
    </script>
  </body>
</html>
